### PR TITLE
PMM-10499: Add datapoints to retrieve node information

### PR DESCRIPTION
**Pull request by dliakhov**
PMM-10499: Add datapoints to retrieve node information

Build: https://github.com/Percona-Lab/pmm-submodules/pull/2710

### CODE REVIEW

File: managed/services/telemetry/config.default.yml [hunk 1]
```diff
@@ -246,6 +246,62 @@ telemetry:
...
+    query: topk(1, node_filesystem_size_bytes{node_name=~"pmm-server",fstype!="fuse.grpcfuse"})
+    summary: "Disk capacity"
+    data:
+      - metric_name: "pmm_server_node_disk_capacity"
+        value: 1
+
+  - id: NodesOSName
+    source: VM
+    query: node_uname_info{node_id=~"pmm-server"}
+    summary: "Disk capacity"
```
Comment (artemgavrilov) [inline managed/services/telemetry/config.default.yml:292]: Wrong summary

File: managed/services/telemetry/config.default.yml [hunk 2]
```diff
@@ -246,6 +246,62 @@ telemetry:
...
+    data:
+      - metric_name: "pmm_server_node_os_name"
+        label: "sysname"
+
+  - id: NodesOSVersion
+    source: VM
+    query: node_uname_info{node_id=~"pmm-server"}
+    summary: "Disk capacity"
```
Comment (artemgavrilov) [inline managed/services/telemetry/config.default.yml:300]: Wrong summary

**Review by artemgavrilov — APPROVED** [review]
LGTM with comments

File: managed/services/telemetry/config.default.yml (hunk 1 above)
Comment (dliakhov) [reply managed/services/telemetry/config.default.yml]: fixed

File: managed/services/telemetry/config.default.yml (hunk 2 above)
Comment (dliakhov) [reply managed/services/telemetry/config.default.yml]: fixed

**Review by ritbl — APPROVED** [review]

**Review by idoqo — APPROVED** [review]

**Pull request merged**